File: src/main/java/ar/edu/um/haberes/rest/model/view/NovedadDuplicada.java
/**
 * 
 */
package ar.edu.um.haberes.rest.model.view;

import java.io.Serializable;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import org.hibernate.annotations.Immutable;

import lombok.Data;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table(name = "vw_novedad_duplicada")
@Immutable
public class NovedadDuplicada implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = -3079597245799350630L;

	@Id
	private String unified;
	
	private Long legajoId;
	private Integer anho;
	private Integer mes;
	private Integer codigoId;
	private Integer cantidad;

}

File: src/main/java/um/haberes/rest/kotlin/model/view/NovedadDuplicada.kt
package um.haberes.rest.kotlin.model.view

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import org.hibernate.annotations.Immutable

@Entity
@Table(name = "vw_novedad_duplicada")
@Immutable
data class NovedadDuplicada(

    @Id
    var unified: String? = null,

    var legajoId: Long? = null,
    var anho: Int? = null,
    var mes: Int? = null,
    var codigoId: Int? = null,
    var cantidad: Int? = null

)
